shell/src/main/java/alluxio/cli/AlluxioShell.java

```diff
@@ -70,19 +65,43 @@ public static void main(String[] argv) throws IOException {
     System.exit(ret);
   }
 
-  private final Map<String, ShellCommand> mCommands;
   private final FileSystem mFileSystem;
 
   /**
    * Creates a new instance of {@link AlluxioShell}.
    */
   public AlluxioShell() {
     mFileSystem = FileSystem.Factory.get();
-    mCommands = AlluxioShellUtils.loadCommands(mFileSystem);
   }
 
   @Override
-  public void close() throws IOException {
+  public int run(String... argv) {
+    if (argv.length > 0) {
+      String cmd = argv[0];
+      String[] replacementCmd = getReplacementCmd(cmd);
+      if (replacementCmd != null) {
+        // Handle command alias, and print out WARNING message for deprecated cmd.
+        String deprecatedMsg = "WARNING: " + cmd + " is deprecated. Please use "
+            + StringUtils.join(replacementCmd, " ") + " instead.";
+        System.out.println(deprecatedMsg);
+        LOG.warn(deprecatedMsg);
```

Comment:
This double logging doesn't seem ideal. Doesn't need to be part of this PR, but what do you think of making shell log to console and only using logging methods?